src/ILCompiler/src/Program.cs

```diff
@@ -356,7 +358,7 @@ private int Run(string[] args)
                     // TODO: Wasm fails to compile some of the xported methods due to missing opcodes
                     if (!_isWasmCodegen)
                     {
-                        compilationRoots.Add(new ExportedMethodsRootProvider((EcmaModule)typeSystemContext.SystemModule));
+                        compilationRoots.Add(new ExportedMethodsRootProvider((EcmaModule)typeSystemContext.SystemModule, _exportsFile));
```

Comment:
Do not pass in `_exportsFile` argument here - pass in null, or make null to be default value of the argument.